torch/csrc/jit/python/pybind_utils.h

```diff
@@ -320,7 +320,7 @@ inline InferredType tryToInferType(py::handle input) {
   if (py::isinstance<Object>(input)) {
     auto object = py::cast<Object>(input);
     return InferredType(object.type());
-#ifdef USE_DISTRIBUTED
+#if defined(USE_DISTRIBUTED) && !defined(_WIN32)
```

Comment:
nit: all these `#if defined(USE_DISTRIBUTED) && !defined(_WIN32)` guards are for RPC. I wonder if we should create a new flag `USE_RPC` in `CMakeLists.txt` and infer it's value using `USE_DISTRIBUTED` and `_WIN32`. Then all the changes in jit-related files can become `#ifdef USE_RPC`.

This can come in a followup PR.